### PR TITLE
Xulkal user changes

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

Created a replacement Tap Dance Double functionality saving another ~700 bytes of firmware space.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/xulkal/process_records.c [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
 
 #if defined(RGB_MATRIX_ENABLE)
 extern void eeconfig_update_rgb_matrix_default(void);
 #endif
 
 #ifdef TRILAYER_ENABLED
-uint32_t layer_state_set_user(uint32_t state) {
-  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+uint32_t layer_state_set_user(uint32_t state)
+{
```
Comment (drashna) [inline users/xulkal/process_records.c]: Why?!

Comment (XScorpion2) [reply users/xulkal/process_records.c]: Because I have a huge monitor

Comment (drashna) [reply users/xulkal/process_records.c]: lol, but .... eeeew

Also, technically:
https://docs.qmk.fm/#/contributing?id=coding-conventions

Comment (XScorpion2) [reply users/xulkal/process_records.c]: ya, but mah keymapz

Comment (drashna) [reply users/xulkal/process_records.c]: lol.  ..... fine. 

But yes, in general, we do want to keep the code style consistent.... as possible.

Though, that's more important for core files, I suppose.

**Pull request merged**